### ISSUE TITLE
In depth learning yolov5 training, the map suddenly drops, so does the adjusted learning rate. What is the problem? Is there a solution?

### ISSUE THREAD

**Issue by ZZM11** [iscyy/ultralyticsPro#147]
![train_loss](https://user-images.githubusercontent.com/63572595/206939179-112cb1c9-902c-4039-a76f-14d38e4619f6.png)
![val_loss](https://user-images.githubusercontent.com/63572595/206939184-9dc3a8e4-eb94-4af6-af60-aaa64d247305.png)

In depth learning yolov5 training, the map suddenly drops, so does the adjusted learning rate. What is the problem? Is there a solution?
The first is map
The second picture is train loss
Page 3 is vai_ loss
Please give me some advice
The two lines are that the learning rate decreases from 0.01 to 0.005
![Snipaste_2022-12-11_20-55-54](https://user-images.githubusercontent.com/63572595/206939185-de0b55f3-0283-413b-a590-fb34c0dbcb1d.png)
